TestConnectionTimeouts fails

Hey, this test has been failing since a couple of merges back:

```
--- FAIL: TestConnectionTimeouts (2.86s)
	dial_test.go:608: hanging dials shouldnt block good dials
```

Not sure how critical this is. Fails since: https://travis-ci.org/libp2p/go-libp2p-conn/builds/341527542

@Stebalien you authored that commit. Any idea?

--- a/dial_test.go
+++ b/dial_test.go
@@ -548,8 +548,8 @@ func TestConnectionTimeouts(t *testing.T) {
 		AcceptTimeout = oldD
 	}(DialTimeout, AcceptTimeout)
 
-	DialTimeout = time.Second * 1
-	AcceptTimeout = time.Second * 1
+	DialTimeout = time.Second * 4
+	AcceptTimeout = time.Second * 4
 
 	p1 := tu.RandPeerNetParamsOrFatal(t)
 

